cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsRdsLaunchService.java

```diff
@@ -105,14 +110,23 @@
         }
 
         List<CloudResource> databaseResources = getCreatedOutputs(ac, stack, cFStackName, cfRetryClient, resourceNotifier);
+        // FIXME: For now, just return everything wrapped in a status object
+        return databaseResources.stream()
+                .map(resource -> new CloudResourceStatus(resource, ResourceStatus.CREATED))
+                .collect(Collectors.toList());
         // FIXME check does nothing?!
-        return awsResourceConnector.check(ac, databaseResources);
+        //return awsResourceConnector.check(ac, databaseResources);
     }
 
     private List<CloudResource> getCreatedOutputs(AuthenticatedContext ac, DatabaseStack stack, String cFStackName, AmazonCloudFormationRetryClient client,
             PersistenceNotifier resourceNotifier) {
         List<CloudResource> resources = new ArrayList<>();
 
+        String hostname = getHostname(cFStackName, client);
+        CloudResource hostnameResource = new Builder().type(ResourceType.RDS_HOSTNAME).name(hostname).build();
+        resourceNotifier.notifyAllocation(hostnameResource, ac.getCloudContext());
```

Comment:
It's possible that this notification isn't necessary, but I'm fine with leaving it.